app/controllers/admin/content_controller.rb

```diff
@@ -37,6 +37,24 @@ def edit
     new_or_edit
   end
 
+  def reverse
+    @article = Article.find(params[:id])
+    if @article != nil
+      text = @article.body
+      split_text = text.split
+      split_text.reverse!
+      reverse_string = ""
+      split_text.each do |word|
```

Comment:
Thanks for your comments. 
I added an if statement to check if there is params[:id] , even though, I can't think of a scenario when it wouldn't have one since the Revers is per post- so it has to have an id, right ?

I removed the each ;) Don't know it that what you meant, but it sure shorter! 
and you right, I did add the db_development and db_test files back in accidentally... Not sure how to re-do it.. can I just delete it??